app/views/articles/show.html.erb

```diff
@@ -190,7 +190,7 @@
 
       </article>
 
-      <% cache("article-bottom-content-#{@article.id}-#{user_signed_in?}-#{(@organization || @user).latest_article_updated_at}", expires_in: 18.hours) do %>
+      <% cache("article-bottom-content-#{rand(5)}-#{@article.id}-#{user_signed_in?}-#{(@organization || @user).latest_article_updated_at}", expires_in: 15.minutes) do %>
```

Comment:
There may have been some discussions around choosing these numbers and grouping it with the article suggestions that I do not have context to, hence please let me know if there were. 

I'm hoping to come up with a less aggressive strategy for caching the display ad whilst we still think about moving to an async loaded display ad. The description in the PR may help give context for these changes but I thought that we could perhaps first separate out the article suggestions from the display ad. My thoughts were that maybe we'd want to update article suggestions less often than the display ads.

In addition, I looked at how we implemented the display ads caching for the sidebar and I thought that adding the `rand(5)` would helps give us that dynamic nature of showing the more successful ads instead of displaying the cached version always which is very unlikely to change unless the article gets updated. 

I also updated the expiry time to 15 minutes instead. We use 5 minutes on the sidebar. 

I'm still getting familiar with our caching strategy and hence would love thoughts from folks.

Reply:
> I also updated the expiry time to 15 minutes instead. We use 5 minutes on the sidebar. Maybe 5 minutes is a better option? Maybe we want to keep that consistency? I'm not really sure what rationale we use to determine these numbers?

I don't have an answer to the rationale used for those numbers but I don't have strong opinions on the "best option" question either.

Given the percentage of traffic (high) that goes into articles I do feel like the 5 minute expiration might be too short and this 15 minute expire timeout is better suited for this area, but this is only backed by gut feeling and not other data from my end. So overall I'm okay with the assumptions made 👍🏼